src/extension.ts

```diff
@@ -205,6 +224,42 @@ export class DaCeVSCode {
         );
 
         // Register necessary commands.
+        this.registerCommand('sdfg.compile', () => {
```

Comment:
Where is the Python part?

Reply:
The Python capability was already present from running SDFGs